src/xdocs/config_imports.xml

```diff
@@ -620,15 +620,26 @@ import android.*;
     </section>
 
     <section name="IllegalImport">
+      <p>Since Checkstyle 3.0</p>
       <subsection name="Description" id="IllegalImport_Description">
-        <p>Since Checkstyle 3.0</p>
         <p>
           Checks for imports from a set of illegal packages. By default, the
           check rejects all <code>sun.*</code> packages since
           programs that contain direct calls to the <code>sun.*</code> packages are <a
           href="https://www.oracle.com/technetwork/java/faq-sun-packages-142232.html">"not
           guaranteed to work on all Java-compatible platforms"</a>. To reject other packages,
-          set property <code> illegalPkgs</code> to a list of the illegal packages.
+          set property <code>illegalPkgs</code> to a list of the illegal packages.
+        </p>
+      </subsection>
+
+      <subsection name="Notes" id="IllegalImport_Notes">
```

Comment:
I just happened to think about this now. We say "See Notes" section after we have already discussed the notes on properties we have not explained yet.

Reply:
Should we rethink the sections? Maybe we need Property Notes and Check Notes depending on the subject of the notes? Or ....

Reply:
Preperty notes below  the property table sounds good for me.

Reply:
be careful with section Notes, there will be multiple notes for different properties, so we might have to make direct links to required Note item. 
I still think having very long and detailed description of property is good, but we review it after xdoc-javadoc sync.